ivy_tests/test_ivy/test_frontends/test_torch/test_blas_and_lapack_ops.py

```diff
@@ -12,6 +17,49 @@
 )
 
 
+# triangular_solve
```

Comment:
Please write tests in the same way that the other tests in this file are implemented.